src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <il class="nav__item">
+            <a class="nav__link is-active" href="#">APPLE</a>
```

Comment:
Don't write content in all caps.
Use normal case and `text-transform: uppercase` in CSS.